docs/tutorials/upgrading-ev3dev-using-shell

```diff
@@ -0,0 +1,24 @@
+---
+title: Upgrading ev3dev Using Shell
+subject: Upgrading
+---
+
```

Comment:
Upgrading can take a long time; sometimes even hours. A warning about this would be nice. You can add formatting to put text in a "warning" alert like this:

```
{% include icon.html type="warning" %}
Alert text goes here
{: .alert .alert-warning}
```